src/nxdoc/nuxeo-server/data-store/data-modeling/schema-fields.md

```diff
@@ -202,6 +202,81 @@ db.default.update(
 )
 ```
 
+## Property Characteristic
```

Comment:
What about `Attributes` instead of `Characteristic`?

Reply:
I use this one to follow the code.

I don't know which one is better, what do you think @efge?

Reply:
Characteristic seems fine to my ear but it's usually used in the plural: Property Characteristics.